### PR TITLE
Add scenario13 with break/fix of the FIP port forwarding

### PR DESCRIPTION
**Pull request by slawqo**
This patch adds new role for the scenario13 which is focused on the floating ip port forwarding.

Scenario first creates "vlan" tenant network and attaches 2 VMs to it. One of the VMs have Floating IP associated (this is distributed) and other have FIP Port Forwarding (this traffic should be centralized). Such configuration is not supported by Neutron thus FIP PF's connection don't work.

As fix for that issue, there is new "geneve" network created and VMs are recreated and connected to that new network.
    
Closes: #OSPRH-6202

### CODE REVIEW

File: roles/scenario13/templates/create_env.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+{{ oc_osp_cmd }} openstack security group create scenario-{{ scenario }}-sg
+{{ oc_osp_cmd }} openstack security group rule create --ingress --protocol icmp scenario-{{ scenario }}-sg
+{{ oc_osp_cmd }} openstack security group rule create --ingress --protocol tcp --dst-port 22 scenario-{{ scenario }}-sg
+{{ oc_osp_cmd }} openstack security group rule create --ingress --protocol tcp --dst-port {{ pf_tcp_port }} scenario-{{ scenario }}-sg
+
+# TODO(slaweq): We can revert that to use OSC to generate ssh keys for us once
+#   we will be using python-openstackclient >= 6.3.0 which contains patch
+#   https://review.opendev.org/c/openstack/python-openstackclient/+/881822
+if type ssh-keygen > /dev/null 2>&1; then
+    yes | ssh-keygen -t ecdsa -f {{ ssh_key_file }} -N "" > /dev/null
```
Comment (cubeek) [inline roles/scenario13/templates/create_env.sh.j2:40]: Nice, I learned a new thing, didn't know about the yes utility :)

File: roles/scenario13/templates/create_env.sh.j2 [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    {{ oc_bin }} -n {{ oc_namespace }} cp {{ ssh_key_file }}.pub openstackclient:/tmp/ssh_key.pub
```
Comment (cubeek) [inline roles/scenario13/templates/create_env.sh.j2:41]: This seems to be "installer" specific - perhaps should be conditional?

File: roles/scenario13/templates/create_env.sh.j2 [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+{{ oc_osp_cmd }} openstack security group create scenario-{{ scenario }}-sg
+{{ oc_osp_cmd }} openstack security group rule create --ingress --protocol icmp scenario-{{ scenario }}-sg
+{{ oc_osp_cmd }} openstack security group rule create --ingress --protocol tcp --dst-port 22 scenario-{{ scenario }}-sg
+{{ oc_osp_cmd }} openstack security group rule create --ingress --protocol tcp --dst-port {{ pf_tcp_port }} scenario-{{ scenario }}-sg
+
+# TODO(slaweq): We can revert that to use OSC to generate ssh keys for us once
+#   we will be using python-openstackclient >= 6.3.0 which contains patch
+#   https://review.opendev.org/c/openstack/python-openstackclient/+/881822
+if type ssh-keygen > /dev/null 2>&1; then
```
Comment (cubeek) [inline roles/scenario13/templates/create_env.sh.j2:39]: I don't understand this condition. ssh-keygen is not installed at some boxes?

File: roles/scenario13/tasks/cleanup.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    {{ oc_osp_cmd }} openstack router remove subnet scenario-{{ scenario }}-router scenario-{{ scenario }}-subnet
+    {{ oc_osp_cmd }} openstack router delete scenario-{{ scenario }}-router
+
+    {{ oc_osp_cmd }} openstack security group delete scenario-{{ scenario }}-sg
+
+    for net in $({{ oc_osp_cmd }} openstack network list | grep scenario-{{ scenario }}-network | awk '{ print $2 }'); do
+      {{ oc_osp_cmd }} openstack network delete $net
+    done
+
+- name: "delete scenario {{ scenario }} working directory"
```
Comment (cubeek) [inline roles/scenario13/tasks/cleanup.yml:30]: Shall we also delete the {{ temp_results_file }} ?

**Review by cubeek — APPROVED** [review]

File: roles/scenario13/templates/create_env.sh.j2 (hunk 2 above)
Comment (slawqo) [reply roles/scenario13/templates/create_env.sh.j2:41]: Done

File: roles/scenario13/templates/create_env.sh.j2 (hunk 3 above)
Comment (slawqo) [reply roles/scenario13/templates/create_env.sh.j2:39]: actually this is not needed, thx

File: roles/scenario13/tasks/cleanup.yml (hunk 4 above)
Comment (slawqo) [reply roles/scenario13/tasks/cleanup.yml:30]: yes, done

File: roles/scenario13/templates/create_env.sh.j2 [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+# TODO(slaweq): We can revert that to use OSC to generate ssh keys for us once
+#   we will be using python-openstackclient >= 6.3.0 which contains patch
+#   https://review.opendev.org/c/openstack/python-openstackclient/+/881822
+yes | ssh-keygen -t ecdsa -f {{ ssh_key_file }} -N "" > /dev/null
+{% if installer == 'podified' %}
+{{ oc_bin }} -n {{ oc_namespace }} cp {{ ssh_key_file }}.pub openstackclient:/tmp/ssh_key.pub
+{{ oc_osp_cmd }} openstack keypair create --public-key /tmp/ssh_key.pub scenario-{{ scenario }}-key
+{% else %}
+{{ oc_osp_cmd }} openstack keypair create --public-key {{ ssh_key_file }} scenario-{{ scenario }}-key
```
Comment (cubeek) [inline roles/scenario13/templates/create_env.sh.j2:44]: should it be `{{ ssh_key_file }}.pub`

**Review by cubeek — CHANGES_REQUESTED** [review]

File: roles/scenario13/templates/create_env.sh.j2 (hunk 5 above)
Comment (slawqo) [reply roles/scenario13/templates/create_env.sh.j2]: yes, thx :)

**Review by cubeek — APPROVED** [review]

**Pull request merged**